WordPress/Classes/Utility/ContextManager.m

```diff
@@ -239,6 +239,7 @@ - (NSPersistentStoreCoordinator *)persistentStoreCoordinator
     };
 
     NSError *error = nil;
+    NSMutableArray *errors = [NSMutableArray array];
```

Comment:
Before this, the call to `migrateDataModelsIfNecessary` might try to do a migration. We should also log any error resulting from that attempt.